eZ/Publish/Core/QueryType/QueryParameterContentViewQueryTypeMapper.php

```diff
@@ -61,13 +61,13 @@ private function extractParameters(ContentView $contentView)
 
     /**
      * @param ContentView $contentView
-     * @param string $queryParameterValue
+     * @param string|string[] $queryParameterValue
      *
      * @return mixed
      */
     private function evaluateExpression(ContentView $contentView, $queryParameterValue)
     {
-        if (substr($queryParameterValue, 0, 2) === '@=') {
+        if (is_string($queryParameterValue) && substr($queryParameterValue, 0, 2) === '@=') {
```

Comment:
ok, then we aim to get this into patch release , so you can adapt for suggestions from @nicolas-bastien to make this even more usable ;)